docs/rest-apis/teamsettings/methods/team-defaults.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+parent: TeamSettings
+title: Team Defaults
```

Comment:
Can you change the title to "Set TeamSettings to Defaults"
I accidentally reset the team settings as I wrongly assumed it would return an object of default values

Reply:
Changed it to "Set TeamSettings to Defaults"